renderdoc/driver/vulkan/vk_core.cpp

```diff
@@ -783,6 +783,23 @@ void WrappedVulkan::CheckPendingCommandBufferCallbacks()
   }
 }
 
+void WrappedVulkan::ClearPendingCommandBufferCallbacks()
+{
+  // This should be called after a FlushQ() call so that all events have signalled, otherwise the
+  // VkPendingSubmissionCompleteCallbacks entry will leak
```

Comment:
Minor nit, but since this code should exclusively only be done during shutdown and should never be re-used elsewhere I'd rather just in-line this at the callsite. Rather than having a function which can't and shouldn't be re-used.